File: R/app/src/main/java/com/example/voiceassistent/phoneapi/Phone.java
package com.example.voiceassistent.phoneapi;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Phone implements Serializable {
    @SerializedName("region")
    @Expose
    public Region region;

    @SerializedName("oper")
    @Expose
    public Operator current;

    public class Region{
        @SerializedName("name")
        @Expose
        public String name;
    }

    public class Operator{
        @SerializedName("name")
        @Expose
        public String name;
    }
}

File: Rkt/app/src/main/java/com/example/voiceassistent/phoneapi/Phone.kt
package com.example.voiceassistent.phoneapi

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Phone : Serializable {
    @SerializedName("region")
    @Expose
    var region: Region? = null

    @SerializedName("oper")
    @Expose
    var current: Operator? = null

    inner class Region {
        @SerializedName("name")
        @Expose
        var name: String? = null
    }

    inner class Operator {
        @SerializedName("name")
        @Expose
        var name: String? = null
    }
}